qcodes/instrument_drivers/Keysight/private/Keysight_344xxA_submodules.py

```diff
@@ -464,8 +465,7 @@ def __init__(self, name: str, address: str, silent: bool = False,
 
         options = self._options()
         self.has_DIG = self.is_34465A_34470A and (
-            'DIG' in options
-            or LooseVersion('A.03') <= LooseVersion(idn['firmware'])
+            "DIG" in options or version.parse("A.03") <= version.parse(idn["firmware"])
```

Comment:
Looks like versions of this form are considdered legacy and also deprecated using packaging. We should look for an alternative way of doing version comparisons in these cases

Reply:
Perhaps one could simply convert letters to their ascii mapping before passing to parse

```
In [39]: verstr = "a.1.0"
    ...: b = []
    ...: for v in verstr:
    ...:     if v.isalpha():
    ...:         b.append(str(ord(v.upper())))
    ...:     else:
    ...:         b.append(v)
    ...: c = "".join(b)
```